solutions/enterprise/catalogValidationValues.json.template

```diff
@@ -0,0 +1,7 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "resource_tags": $TAGS,
+  "prefix": $PREFIX,
+  "resource_group_name": $PREFIX,
+  "existing_kms_key_crn": $HPCS_US_SOUTH_ROOT_KEY_CRN
```

Comment:
I suggest to pass a value for `existing_kms_instance_crn` instead of passing the key, so the DA will create a new key

Reply:
done